bundles/org.openhab.binding.sony/README-SimpleIp.md

```diff
@@ -0,0 +1,122 @@
+2.  The input can be either "TV" or "xxxxyyyy" where xxxx is the port name and yyyy is the port number.  Valid port names (case insensitive) are "hdmi", "scart", "composite", "component", "screen mirroring", and "pc rgb input".  The port number is dependent on how many ports the device supports.  Example: the X830 supports 4 hdmi ports - so "hdmi1", "hdmi2", "hdmi3" and "hdmi4" are all valid.
+
+## Full Example
+
+simpleip.Things:
+
+```
+sony:simpleip:home [ deviceAddress="192.168.1.72", commandsMapFile="braviaircodes.map", netInterface="eth0" ]
+```
+
+simpleip.items:
+
+```
+String Bravia_IR "IR [%s]" { channel="sony:simpleip:home:ir" }
+Switch Bravia_Power "Power [%s]" { channel="sony:simpleip:home:power" }
+Dimmer Bravia_Volume "Volume [%s]" { channel="sony:simpleip:home:volume" }
+Switch Bravia_AudioMute "Audio Mute [%s]" { channel="sony:simpleip:home:audiomute" }
+String Bravia_Channel "Channel [%s]" { channel="sony:simpleip:home:channel" }
+String Bravia_TripletChannel "Triplet Channel [%s]" { channel="sony:simpleip:home:tripletchannel" }
+String Bravia_InputSource "Input Source [%s]" { channel="sony:simpleip:home:inputsource" }
+String Bravia_Input "Input [%s]" { channel="sony:simpleip:home:input" }
+Switch Bravia_PictureMute "Picture Mute [%s]" { channel="sony:simpleip:home:picturemute" }
+Switch Bravia_TogglePictureMute "Toggle Picture Mute [%s]" { channel="sony:simpleip:home:togglepicturemute", autoupdate="false"  }
+Switch Bravia_Pip "PIP [%s]" { channel="sony:simpleip:home:pip" }
+Switch Bravia_TogglePip "Toggle PIP [%s]" { channel="sony:simpleip:home:togglepip", autoupdate="false" }
+Switch Bravia_TogglePipPosition "Toggle PIP Position [%s]" { channel="sony:simpleip:home:togglepipposition", autoupdate="false" }
+```
+
+simpleip.sitemap:
+
+```
+sitemap demo label="Main Menu"
+{
+    Frame label="Sony Bravia" {
+        Selection item=Bravia_IR mappings=[Channel-Up="Channel Up",Channel-Down="Channel Down",Left="Left"]
+        Switch item=Bravia_Power
+        Slider item=Bravia_Volume
+        Switch item=Bravia_AudioMute
+        Selection item=Bravia_Channel mappings=[4.1="ABC(1)", 5.1="NBC(1)", 5.2="NBC(2)", 13="CBS", 50.1="WRAL(1)", 50.2="WRAL(2)"]
+        Text item=Bravia_TripletChannel
+        Selection item=Bravia_InputSource mappings=[atsct="ATSCT", dvbt="DVBT", dvbc="DVBC", dvbs="DVBS", isdbt="ISDBT", isdbbs="ISDBBS", isdbcs="ISDBCS", antenna="Antenna", cable="Cable", isdbgt="ISDBGT"]
+        Selection item=Bravia_Input mappings=[TV="TV", HDMI1="HDMI1", HDMI2="HDMI2"]
+        Switch item=Bravia_PictureMute
+        Switch item=Bravia_TogglePictureMute mappings=[ON="Toggle"]
+        Switch item=Bravia_Pip
+        Switch item=Bravia_TogglePip mappings=[ON="Toggle"]
+        Switch item=Bravia_TogglePipPosition mappings=[ON="Toggle"]
+        Text item=Bravia_BroadcastAddress
+        Text item=Bravia_MacAddress
```

Comment:
these two items aren't listed in the sample above

Reply:
Fixed